### DISCUSSION TITLE
Help setting up email notification.

### DISCUSSION THREAD

**Discussion by GenesisAdam** [crazy-max/diun#862] [Q&A] [answered]
I am unable to get the email notification working using docker compose. Looks like I am missing something. The installation instructions are not very clear on using docker compose to set up diun especially where to put the email templates.
I have set up a dium.yaml:
```notif:
  mail:
    host: mail.ebony.com
    port: 25
    ssl: false
    insecureSkipVerify: false
    from: ory@ebony.com
    to: oxx@gmail.com
    templateTitle: "{{ .Entry.Image }} released"
    templateBody: |
      Docker tag {{ .Entry.Image }} which you subscribed to through {{ .Entry.Provider }} provider has been released.
```
and my docker compose file:
```version: "3.5"

services:
  diun:
    image: crazymax/diun:latest
    container_name: diun
    command: serve
    volumes:
      - "/volume1/docker/diun/data:/data"
      - "/volume1/docker/diun/diun.yaml:/diun.yaml"
      - "/var/run/docker.sock:/var/run/docker.sock"
    environment:
      - "TZ=Europe/London"
      - "LOG_LEVEL=info"
      - "LOG_JSON=false"
      - "DIUN_WATCH_WORKERS=20"
      - "DIUN_WATCH_SCHEDULE=0 */6 * * *"
      - "DIUN_PROVIDERS_DOCKER=true"
      - "DIUN_PROVIDERS_DOCKER_WATCHBYDEFAULT=true"
    labels:
      - "diun.enable=true"
      - "diun.watch_repo=true"
    restart: always`
```
I am getting this error messages in the log file:

_today at 13:17  Fri, 30 Jul 2021 13:17:23 BST INF Starting Diun version=4.17.0
today at 13:17  Fri, 30 Jul 2021 13:17:23 BST FTL Cannot load configuration error="Failed to decode configuration from file: field not found, node: templateBody"_

Many thanks for any help.

**Comment by crazy-max** [comment]
Please format your post correctly: https://guides.github.com/features/mastering-markdown/. Thanks.

**Comment by GenesisAdam** [comment]
> 
> 
> Please format your post correctly: https://guides.github.com/features/mastering-markdown/. Thanks.

The post has been formatted. Are you able to help?
Thanks.

**Comment by crazy-max** [comment, marked as answer]
@GenesisAdam `templateTitle` and `templateBody` is available since 4.19.0 and you use 4.17.0: https://crazymax.dev/diun/changelog/. Upgrade to the latest release.

**Comment by GenesisAdam** [comment]
@crazy-max  Thanks. I will try that and see if it resolves the issue for me.
